.eslintrc

```diff
@@ -7,25 +7,38 @@
     {
       "extends": ["eslint:recommended", "plugin:@typescript-eslint/strict", "plugin:lit/recommended"],
       "files": ["*.ts"],
-      "parserOptions": {
-        "createDefaultProgram": true
-      },
       "rules": {
         "@typescript-eslint/no-explicit-any": "off",
-        "@typescript-eslint/no-inferrable-types": "error"
-        // "max-len": ["warn", { "code": 80, "ignoreComments": true }]
+        "@typescript-eslint/no-inferrable-types": "error",
+        // "max-len": ["warn", { "code": 120 }]
+        "no-console": [
+          "error",
+          {
+            // TODO: Remove "debug" from this allowed list
+            "allow": ["warn", "error",  "time", "timeEnd", "debug"]
+          }
+        ]
       },
       "settings": {
         "polyfills": ["navigator.userAgentData"]
       }
     },
     {
-      "files": ["*.spec.ts", "*.fixture.ts"],
+      "extends": "plugin:playwright/recommended",
+      "files": ["*.spec.ts", "*.fixture.ts", "src/tests/**/*.ts"],
+      "excludedFiles": ["node_modules/", "dist/", "@types/"],
       "parserOptions": {
         "project": "./tsconfig.spec.json"
       },
       "rules": {
-        "@typescript-eslint/no-floating-promises":"error"
+        "@typescript-eslint/no-floating-promises":"error",
+        "playwright/valid-describe-callback": "off",
+        "playwright/valid-title": "off",
+        "playwright/expect-expect": "off",
+        "playwright/no-skipped-test": "off",
+        "playwright/no-conditional-in-test": "off",
+        "playwright/no-conditional-expect": "off",
+        "playwright/no-networkidle": "off"
```

Comment:
maybe leave a comment for why this has been disabled?

Reply:
![image](https://github.com/user-attachments/assets/af2b00df-37bf-48a5-9ec9-417c168fab54)